databricks/koalas/groupby.py

```diff
@@ -0,0 +1,321 @@
+        """
+        Compute mean of groups, excluding missing values.
+
+        Returns
+        -------
+        koalas.Series or koalas.DataFrame
+
+        See Also
+        --------
+
+        koalas.DataFrame.groupby
+
+        Examples
+        --------
+        >>> df = ks.DataFrame({'A': [1, 1, 2, 1, 2],
+        ...                    'B': [np.nan, 2, 3, 4, 5],
+        ...                    'C': [1, 2, 1, 1, 2]}, columns=['A', 'B', 'C'])
+
+        Groupby one column and return the mean of the remaining columns in
+        each group.
+
+        >>> df.groupby('A').mean()  # doctest: +NORMALIZE_WHITESPACE
+             B         C
+        A
+        1  3.0  1.333333
+        2  4.0  1.500000
+        """
+
+        return self._reduce_for_stat_function(F.mean, only_numeric=True)
+
+    def min(self):
+        """
+        Compute min of group values.
+
+        See Also
+        --------
+
+        koalas.DataFrame.groupby
+        """
+        return self._reduce_for_stat_function(F.min, only_numeric=False)
+
+    # TODO: sync the doc and implement `ddof`.
+    def std(self):
+        """
+        Compute standard deviation of groups, excluding missing values.
+        """
+
+        return self._reduce_for_stat_function(F.stddev, only_numeric=True)
+
+    def sum(self):
+        """
+        Compute sum of group values
+
+        See Also
+        --------
+
+        koalas.DataFrame.groupby
+        """
+        return self._reduce_for_stat_function(F.sum, only_numeric=True)
+
+    # TODO: sync the doc and implement `ddof`.
+    def var(self):
+        """
+        Compute variance of groups, excluding missing values.
+        """
+        return self._reduce_for_stat_function(F.variance, only_numeric=True)
+
+    def _reduce_for_stat_function(self, sfun, only_numeric):
+        groupkeys = self._groupkeys
+        groupkey_cols = [s._scol.alias('__index_level_{}__'.format(i))
+                         for i, s in enumerate(groupkeys)]
+        sdf = self._kdf._sdf
+
+        column_fields = []
+        if len(self._agg_columns) > 0:
+            stat_exprs = []
+            for ks in self._agg_columns:
+                spark_type = ks.spark_type
+                # Special handle floating point types because Spark's count treats nan as a valid
+                # value, whereas Pandas count doesn't include nan.
+                if isinstance(spark_type, DoubleType) or isinstance(spark_type, FloatType):
+                    stat_exprs.append(sfun(F.nanvl(ks._scol, F.lit(None))).alias(ks.name))
+                    column_fields.append(ks.name)
+                elif isinstance(spark_type, NumericType) or not only_numeric:
+                    stat_exprs.append(sfun(ks._scol).alias(ks.name))
+                    column_fields.append(ks.name)
+            sdf = sdf.groupby(*groupkey_cols).agg(*stat_exprs)
+        else:
+            sdf = sdf.select(*groupkey_cols).distinct()
+        sdf = sdf.sort(*groupkey_cols)
```

Comment:
@HyukjinKwon Do you remember why we added `sort` here?

Reply:
It was from your PR (https://github.com/databricks/koalas/pull/194/files#diff-611bb17e37caad740e4b9943a69c3b4bR97) :-) ..

Reply:
Yeah, right. I don't remember the reason. haha

Reply:
Looks we can remove